packages/esm-patient-registration-app/src/patient-registration/field/address/address-hierarchy.component.tsx

```diff
@@ -22,31 +23,49 @@ export const AddressHierarchy: React.FC = () => {
   const [addressLayout, setaddressLayout] = useState([]);
   const { t } = useTranslation();
   const { addressTemplate } = useContext(ResourcesContext);
-  const addressTemplateXml = addressTemplate.results[0].value;
+  const addressTemplateXml = addressTemplate?.results[0].value;
 
   const setSelectedValue = (value: string) => {
     setSelected(value);
   };
 
   useEffect(() => {
-    const templateXmlDoc = parseString(addressTemplateXml);
-    const elementDefaults = getTagAsDocument('elementdefaults', templateXmlDoc);
-    const nameMappings = getTagAsDocument('nameMappings', templateXmlDoc);
-    const properties = nameMappings.getElementsByTagName('entry');
-    const propertiesObj = Array.prototype.map.call(properties, (property: Element) => {
-      const name = property.getElementsByTagName('string')[0].innerHTML;
-      const labelText = t(name, property.getElementsByTagName('string')[1].innerHTML);
-      const value = getFieldValue(name, elementDefaults);
-      return {
-        id: name,
-        name,
-        labelText,
-        value,
-      };
-    });
-    setaddressLayout(propertiesObj);
+    if (addressTemplateXml) {
+      const templateXmlDoc = parseString(addressTemplateXml);
+      const elementDefaults = getTagAsDocument('elementdefaults', templateXmlDoc);
+      const nameMappings = getTagAsDocument('nameMappings', templateXmlDoc);
+      const properties = nameMappings.getElementsByTagName('entry');
+      const propertiesObj = Array.prototype.map.call(properties, (property: Element) => {
+        const name = property.getElementsByTagName('string')[0].innerHTML;
+        /*
```

Comment:
I know that this is not your code, but would it be possible for you to (again) add a comment here? If I didn't know about the purpose, I'd probably remove it because it looks like dead code. 😄 

`Please do not remove this comment. It exists to force the translation keys to be generated.`

Reply:
Thanks @manuelroemer!